tests/FunctionalClientTest.php

```diff
@@ -4,7 +4,7 @@
 
 use Clue\React\Block;
 use Clue\React\Docker\Client;
-use React\EventLoop\Factory as LoopFactory;
+use React\EventLoop\Loop as LoopFactory;
```

Comment:
Alias should be updated?

Reply:
I removed it, I don't see that it's still needed